index.html

```diff
@@ -95,16 +45,70 @@ <h2 class='no-margin-top'>
         <a class='btn btn-link btn-lg' href="{{post.agenda}}" target='_blank'>Agenda</a>
       </div>
       <div class='col-sm-5'>
-        {% if post.image %}
+        {% if post.image and post.description.size > 0 %}
           <a href="{{post.url}}"><img class='img-responsive img-thumbnail' src='{{post.image}}' alt='{{post.title}}' /></a>
           {% if post.image_credit %}
             <br /><small class='pull-right'><em>Photo: {{ post.image_credit }}</em></small>
           {% endif %}
         {% endif %}
       </div>
     </div>
+      
+  </div>
+</div>
+{% endfor %}
+
+{% assign past_events = site.categories['events'] | where_exp:"post", "post.date < site.time" %}
+{% for post in past_events limit: 1 %}
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title"><b>Last Chi Hack Night</b> <a class='pull-right' href='/events/'>Past events &raquo;</a></h3>
+  </div>
+  <div class="panel-body">
+    <strong>{{ post.date | date: "%l:%M%P" }} {{ post.date | date: "%A, %B %-d, %Y" }}</strong>
+    <h2 class='no-margin-top'>
+      <small>#{{post.event_id}}</small>
+      <a href="{{post.url}}">{{post.title}}</a>
+    </h2>
+    <div class='row'>
+      <div class='col-sm-12'>
+        {% if post.description %}
+          <p>{{ post.description }}</p>
+        {% endif %}
+        <a class='btn btn-link btn-lg' href="{{post.url}}"><i class='fa fa-info-circle-square-o'></i> Details</a>
+        {% if post.youtube_id %}
+        <a class='btn btn-link btn-lg' href="https://www.youtube.com/watch?v={{post.youtube_id}}" target='_blank'><i class='fa fa-play-circle'></i> YouTube Video</a>
+        {% endif %}
+      </div>
+      <!-- <div class='col-sm-5'>
```

Comment:
should we delete this commented section?

Reply:
Done https://github.com/chihacknight/chihacknight.org/pull/389/commits/6ffeb4f38c55805f9f36f21e2fe4dfc10a7b1b87